modules/dbnd-airflow/src/dbnd_airflow/airflow_extensions/request_factory.py

```diff
@@ -1,38 +1,71 @@
-from airflow.contrib.kubernetes.kubernetes_request_factory.pod_request_factory import (
-    SimplePodRequestFactory as AirflowSimplePodRequestFactory,
-)
+import logging
+import pprint
+
+from distutils.version import LooseVersion
+
+import airflow
+import yaml
+
 from airflow.contrib.kubernetes.secret import Secret
 
 
-class DbndPodRequestFactory(AirflowSimplePodRequestFactory):
+logger = logging.getLogger(__name__)
+
+
+class DbndPodRequestFactory(object):
+    def __init__(self, kubernetes_engine_config, pod_yaml):
+        self.kubernetes_engine_config = kubernetes_engine_config
+        self.yaml = yaml.safe_load(pod_yaml) or {}
+
     def create(self, pod):
-        req = super(DbndPodRequestFactory, self).create(pod=pod)
-        self.extact_extended_resources(pod, req)
+        original_pod = pod
+
+        if LooseVersion(airflow.version.version) > LooseVersion("1.10.10"):
```

Comment:
We use this in few places, would it make sens to introduce `AIRFLOW_ABOVE_10 = LooseVersion(airflow.version.version) > LooseVersion("1.10.10")`?

Reply:
yes, definetly